integration-tests/gatsby-pipeline/__tests__/lazy-image-build/develop.js

```diff
@@ -0,0 +1,61 @@
+const execa = require(`execa`)
+const path = require(`path`)
+const fs = require(`fs-extra`)
+const glob = require(`glob`)
+const request = require(`request-promise-native`)
+const createDevServer = require(`../../utils/create-devserver`)
+const basePath = path.resolve(__dirname, `../../`)
+
+// 2 min
+jest.setTimeout(2000 * 60)
+
+const cleanDirs = () =>
+  Promise.all([
+    fs.emptyDir(`${basePath}/public`),
+    fs.emptyDir(`${basePath}/.cache`),
+  ])
+
+describe(`Lazy images`, () => {
+  beforeAll(async () => {
+    await cleanDirs()
+  })
+
+  test(`should process images on demand`, async () => {
+    const { kill } = await createDevServer()
+
+    const response = await request(
+      `http://localhost:8000/static/6d91c86c0fde632ba4cd01062fd9ccfa/a2541/gatsby-astronaut.png`,
+      {
+        resolveWithFullResponse: true,
```

Comment:
This grabs the statusCode right? Not just the response body?

Reply:
you get the full response yes, I'll add a comment :)

Reply:
Oh no - it's OK! Was just so I know--I hadn't seen that before!